### PR TITLE
Add Ripe cloud mode docs

### PR DESCRIPTION
**Pull request by simpajj**
### Proposed changes

Added documentation for the Ripe cloud mode integration and updated the documentation for the device mode integration accordingly. See the cloud mode PR for reference: https://github.com/segmentio/action-destinations/pull/952

The device mode integration has been renamed to `Ripe Web Destination (Actions)` for clarity, and the new cloud mode one is called `Ripe Cloud Destination (Actions)`.

### Merge timing
- ASAP once approved

### Related issues (optional)

https://github.com/segmentio/action-destinations/pull/952

### CODE REVIEW

File: src/connections/destinations/catalog/actions-ripe-cloud/index.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+title: Ripe Cloud Destination (Actions)
```
Comment (stayseesong) [inline src/connections/destinations/catalog/actions-ripe-cloud/index.md:2]:
```suggestion
title: Ripe Cloud (Actions) Destination 
```

File: src/connections/destinations/catalog/actions-ripe-web/index.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Ripe Destination (Actions)
+title: Ripe Web Destination (Actions)
```
Comment (stayseesong) [inline src/connections/destinations/catalog/actions-ripe-web/index.md:2]:
```suggestion
title: Ripe Web (Actions) Destination 
```

**Review by stayseesong — CHANGES_REQUESTED** [review]

**Comment by stayseesong** [comment]
@simpajj I'll merge this in once the source is available in the catalog

**Comment by markzegarelli** [comment]
Moved this to #4118

**Pull request closed without merging**